db/model/room.js

```diff
@@ -33,6 +34,14 @@ module.exports = function room(seq, dataTypes) {
       },
       comment: 'Create a named room ',
     },
+    settings: {
+      type: dataTypes.ARRAY(dataTypes.JSONB),
+      allowNull: true,
+      validate: {
+        contains: u.validateSettingsArray,
+      },
+      comment: 'Key/Value pairs for user specific settings',
```

Comment:
Can you and the IMC team discuss this design for storing these key/value pairs--I'd like to see a uniform approach consistent across the different models where applicable, but I'm not sure an array is the right way to store it. I'd rather get it right the first time, because field changes require a migration.